app/src/main/java/me/skrilltrax/themoviedb/di/NetworkModule.kt

```diff
@@ -0,0 +1,63 @@
+package me.skrilltrax.themoviedb.di
+
+import com.github.ajalt.timberkt.Timber.d
+import dagger.Module
+import dagger.Provides
+import dagger.hilt.InstallIn
+import dagger.hilt.android.components.ApplicationComponent
+import javax.inject.Singleton
+import me.skrilltrax.themoviedb.constants.Constants
+import me.skrilltrax.themoviedb.network.api.movie.MovieApiInterface
+import me.skrilltrax.themoviedb.network.api.tv.TVApiInterface
+import okhttp3.OkHttpClient
+import okhttp3.logging.HttpLoggingInterceptor
+import retrofit2.Retrofit
+import retrofit2.converter.gson.GsonConverterFactory
+
+@InstallIn(ApplicationComponent::class)
+@Module
+object NetworkModule {
+
+    @Provides
+    @Singleton
```

Comment:
Shouldn't `ApplicationComponent` imply this?

Reply:
```Warning: A common misconception is that all bindings declared in a module will be scoped to the component the module is installed in. However, this isn’t true. Only bindings declarations annotated with a scope annotation will be scoped.```

This is from https://dagger.dev/hilt/components